docs/userguides/platform.md

```diff
@@ -35,7 +35,41 @@ Building a container for your application can be an advanced topic, we have incl
 
 ## Building your Bot
 
-TODO: Add build process and describe `silverback build --autogen` and `silverback build --upgrade`
+To build your container definition(s) for your bot(s), you can use the `silverback build` command. This command searches your `bots` directory for python modules, then auto-generates Dockerfiles.
```

Comment:
```suggestion
To create Docker container images for your bots, you can use the `silverback build` command.
This command searches for your bots according to the heuristic above in your project directory, and then auto-generates `Dockerfile`s for each bot it finds.
```

Reply:
Wonder if this should end up under like an `images/` or `.images/` folder (or maybe `.silverback-images`?), so it doesn't pollute the root directory?

Reply:
we can do this, I'm working on it now. It just changes how you build the image.